pkg/util/helm/helm.go

```diff
@@ -139,3 +139,13 @@ func GetAnnotationName() string {
 func GetLabelName() string {
 	return "app.kubernetes.io/instance"
 }
+
+func GetBoolTrueAddress() *bool {
+	trueValue := true
+	return &trueValue
+}
+
+func GetBoolFalseAddress() *bool {
+	falseValue := false
+	return &falseValue
+}
```

Comment:
https://github.com/google/go-github/blob/master/github/github.go#L1352-L1366

We can reuse these functions that convert type to pointers by placing them in `pkg/util/types`